src/controllers/signUp.js

```diff
@@ -1,6 +1,66 @@
-exports.get = (req,res) => {
-    res.render("sign-up", {
-        title: "Sign-up", headerFound: false, footerFound: false, asideFound:false, style: ["signUpIn"],
-         javascript: ['sign-up']
+const bcrypt = require('bcryptjs');
```

Comment:
Kannan can you please divide the hassing into separate file in case if using it a gain in the adding doctor in the admin section